src/main/java/com/example/garage/service/CarService.java

```diff
@@ -0,0 +1,61 @@
+package com.example.garage.service;
+
+import com.example.garage.model.Car;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Getter
+@Setter
+public class CarService {
+
+    public static List<Car> cars = new ArrayList<>();
+
+    @Autowired
+    public CarService(List<Car> cars) {
+        CarService.cars = cars;
+    }
+
+    int carCounter = 1;
+
+    public Car createCar(Car car) {
```

Comment:
it is CarService so it works only with cars.  createCar -> create  and the same strategy for all methods below